File: Basics_code_igenuine/java/works/fibo.java
import java.util.Scanner;
public class fibo
{
	public static void main(String[]args)
	{
		int i;
		int n,t1,t2,nxt;
		t1=0;
		t2=1;
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter the count");
		n=sc.nextInt();
		sc.close();
		for(i=1;i<=n;i++)
		{
			System.out.print(t1+" ");
			nxt=t1+t2;
			t1=t2;
			t2=nxt;
		}
	}
}

File: Basics_code_igenuine/php/myweb/works/fibo.php
<?php
	function fibo($n)
	{
		$t1=0;
		$t2=1;
		for($i=1;$i<=$n;$i++)
		{
			echo $t1." ";
			$nxt=$t1+$t2;
			$t1=$t2;
			$t2=$nxt;
		}
	}
?>
<html>
<head></head>
<body>
<?php
if(!isset($_POST['submit']))
{
?>
<form method="post" action="fibo.php">
Enter the count for fibonacci Series :<br>
<input type="text" name="count"size="4"/><br>
<input type="submit" name="submit"value="Submit"/><br>
</form>
<?php
}
else
{
	$num=(int)$_POST['count'];
	echo fibo($num); 
	
}
?>